card/api/card.api

```diff
@@ -161,34 +161,6 @@ public final class com/adyen/checkout/card/BuildConfig {
 	public fun <init> ()V
 }
 
-public final class com/adyen/checkout/card/CardBrand : android/os/Parcelable {
```

Comment:
This is a breaking change, let's add a type alias and test that all public functionality is not affected?